### PR TITLE
correct path to routegroups

### PR DESCRIPTION
**Pull request by adyach**
a route groups link was pointing in to one folder up

### CODE REVIEW

File: docs/kubernetes/ingress-usage.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ This documentation is meant for people deploying to Kubernetes
 Clusters and describes to use Ingress and low level and high level
 features Skipper provides.
 
-[RouteGroups](../routegroups/), a relatively new feature, also
+[RouteGroups](routegroups.md/), a relatively new feature, also
```
Comment (AlexanderYastrebov) [inline docs/kubernetes/ingress-usage.md:7]: Slash at the end is redundant

File: docs/kubernetes/ingress-usage.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ This documentation is meant for people deploying to Kubernetes
 Clusters and describes to use Ingress and low level and high level
 features Skipper provides.
 
-[RouteGroups](../routegroups/), a relatively new feature, also
+[RouteGroups](routegroups.md), a relatively new feature, also
 support each of these features, with an alternative format that
 supports them in a more native way. The documentation contains a
 section with
-[mapping](../routegroups/#mapping-from-ingress-to-routegroups)
+[mapping](routegroups.md/#mapping-from-ingress-to-routegroups)
```
[the remark this answers is not in the document]
Comment (AlexanderYastrebov) [reply docs/kubernetes/ingress-usage.md:11]: I think there is something else going on that relates to https://opensource.zalando.com/skipper/kubernetes/ingress-usage/ - there links work as expected as far as I can tell. 

@szuecs What is the correct way to use links? I see a benefit of having navigable github links as well, can we make them work for both? WDYT?

[the remark this answers is not in the document]
Comment (AlexanderYastrebov) [reply docs/kubernetes/ingress-usage.md:11]: I think it is safe to use *.md, e.g.
https://github.com/zalando/skipper/blame/5727482869e227691446b55f5de4189ea2d5e9df/docs/reference/filters.md#L1597
links `scripts.md` and it looks/links ok here https://opensource.zalando.com/skipper/reference/scripts/ and here https://opensource.zalando.com/skipper/reference/filters/#lua

**Comment by szuecs** [comment]
:+1:

File: docs/kubernetes/ingress-usage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (szuecs) [reply docs/kubernetes/ingress-usage.md]: Yeah back then I was not aware of the magic .md links that work best, which might be the reason why browsing markdown files in the repository does not work as expected

**Comment by adyach** [comment]
👍

**Comment by AlexanderYastrebov** [comment]
I have created a followup #1741

**Pull request merged**